timemachine/fe/atom_mapping.py

```diff
@@ -265,42 +265,22 @@ def induce_mol_subgraph(mol_a, core_a, bond_core_a):
     return sg_a
 
 
-def _compute_bond_cores(mol_a, mol_b, marcs):
```

Comment:
I've noticed a couple of other functions in this module seem unused: `bfs`, `find_cycles`, `induce_mol_subgraph`, `update_bond_core`. Would it make sense to also clean these up in this PR?

Reply:
Removed in https://github.com/proteneer/timemachine/pull/1362/commits/807a91ca14b53803fb91bc0a41fe648a7fa89858